pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -1196,6 +1196,55 @@ func (c *Reconciler) updatePipelineRunStatusFromInformer(ctx context.Context, pr
 	return nil
 }
 
+// filterTaskRunsForPipelineRun takes a slice of TaskRuns with the given PipelineRun marked as owner via the "tekton.dev/pipelineRun"
+// label, and returns a slice of TaskRuns directly specified in the PipelineRun and a map of condition TaskRuns keyed by
+// the PipelineTask name they relate to.
```

Comment:
This docstring could be rewritten as "filterTaskRunsForPipelineRun returns TaskRuns owned by the PipelineRun and their condition checks"

Reply:
👍